rustfmt.toml

```diff
@@ -3,3 +3,4 @@ normalize_doc_attributes = true
 wrap_comments = true
 format_strings = true
 use_field_init_shorthand = true
+imports_layout = "HorizontalVertical"
```

Comment:
I'm not sure we want this one. Use statements can then easily exceed en entire display worth of locs.

Reply:
Yeah I'm on the fence as well. On the other hand diffs become much easier to read because things aren't flying around everywhere.

Reply:
Other options: :ok:

Reply:
>diffs become much easier to read because things aren't flying around everywhere.

Ok, this actually makes a difference. Better diffing > locs.

Reply:
We can always experiment and see - easy enough to change now.